IRremoteESP8266.cpp

```diff
@@ -470,7 +470,7 @@ void ICACHE_FLASH_ATTR IRsend::sendRCMM(uint32_t data, uint8_t nbits) {
 void ICACHE_FLASH_ATTR IRsend::sendPanasonic64(unsigned long long data,
                                                unsigned int nbits,
                                                unsigned int repeat) {
-  enableIROut(37);  // Set IR carrier frequency
+  enableIROut(36700U);  // Set IR carrier frequency of 36.7kHz.
   IRtimer usecTimer = IRtimer();
 
   for (uint16_t i = 0; i <= repeat; i++) {
```

Comment:
I am sorry, I should have been more specific. The loop did not appear to execute any statements so I thought it was redundant. Now, upon closer inspection I realized that github beautifully collapsed the code within the loop and I failed to notice the jump in line numbers. :no_mouth: